pkg/eval/str/str.go

```diff
@@ -92,6 +95,62 @@ import (
 // ▶ $false
 // ```
 
+//elvdoc:fn from-codepoints
+//
+// ```elvish
+// str:from-codepoints $number...
+// ```
+//
+// Outputs a string consisting of the given Unicode codepoints. Example:
+//
+// ```elvish-transcript
+// ~> str:from-codepoints 0x61
+// ▶ a
+// ~> str:from-codepoints 0x4f60 0x597d
+// ▶ 你好
+// ```
+//
+// @cf str:to-codepoints
+
+func fromCodepoints(nums ...int) (string, error) {
+	var b bytes.Buffer
+	for _, num := range nums {
```

Comment:
Check if num is within the range of rune, and return an [OutOfRange](https://github.com/elves/elvish/blob/9858d5a5455ab157934d84b118a4d63c45dbfcd4/pkg/eval/errs/errs.go#L9) error if it isn't. Also add a test for this.